packages/gatsby-cli/src/init-starter.ts

```diff
@@ -183,7 +189,7 @@ const clone = async (hostInfo: any, rootPath: string) => {
   if (!isGit) await createInitialGitCommit(rootPath, url)
 }
 
-const getPaths = async (starterPath: string, rootPath: string) => {
+const getPaths = async (starterPath: string, rootPath?: string) => {
```

Comment:
looks like starterPath is optional too? According to L196. If it's always passed it in it's usages maybe we drop the condition on it